src/components/JobsPanelResources/JobsPanelResourcesView.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import JobsPanelSection from '../../elements/JobsPanelSection/JobsPanelSection'
+import Select from '../../common/Select/Select'
+import RangeInput from '../../common/RangeInput/RangeInput'
+
+const JobsPanelResourcesView = ({
+  cpuUnit,
+  edit,
+  handleSelectСpuUnit,
+  handleSelectMemoryUnit,
+  limits,
+  match,
+  memoryUnit,
+  requests,
+  requestsCpu,
+  requestsMemory,
+  setLimits,
+  setRequests
+}) => (
+  <div className="job-panel__item resources">
+    {!edit && <div className="item__overlay" />}
+    <JobsPanelSection title="Resources" />
+    <JobsPanelSection title="Memory">
+      <Select
+        match={match}
+        option="unitMemory"
+        onClick={value => handleSelectMemoryUnit(value)}
+        label="Unit"
+        value={memoryUnit}
+      />
+      <RangeInput
+        onChange={value =>
+          setRequests({
+            ...requests,
+            memory: `${value}${
+              memoryUnit.length > 0 && memoryUnit !== 'Bytes'
+                ? memoryUnit.match(/i/)
+                  ? memoryUnit.slice(0, 2)
+                  : memoryUnit.slice(0, 1)
+                : ''
```

Comment:
Maybe this will be easier to read:
```suggestion
              memoryUnit.length === 0 || memoryUnit === 'Bytes'
              ? ''
              : memoryUnit.match(/i/)
                ? memoryUnit.slice(0, 2)
                : memoryUnit.slice(0, 1)
```